cmd/influxd/config.go

```diff
@@ -49,13 +51,6 @@ type Config struct {
 		Assets string `toml:"assets"`
 	} `toml:"admin"`
 
-	HTTPAPI struct {
-		Port        int      `toml:"port"`
-		SSLPort     int      `toml:"ssl-port"`
```

Comment:
I think we hope to support encrypted data transfers at some point, we'll have to add this stuff back in eventually.